template/package.json

```diff
@@ -4,9 +4,9 @@
   "description": "<%= description %>",
   "main": "index.js",
   "scripts": {
-    "start": "budo . -d --serve bundle.js --live",
-    "build": "browserify . -o bundle.js -g uglifyify",
-    "test": "browserify test.js | smokestack | tap-spec",
+    "start": "budo . -t babelify -d --serve bundle.js --live",
```

Comment:
you should only add `babelify` to the `browserify.transform` options within the rest of your `package.json`, as so:

```json
{
  ...
  "browserify": {
    "transform": [
      "babelify"
    ]
  },
  ...
}
```

this way you keep your config DRY.